overrides.nix

```diff
@@ -103,6 +103,13 @@ self: super: {
     ename = "so-long";
   };
 
+  tree-sitter = super.tree-sitter.overrideAttrs (esuper: {
+    postInstall = ''
+      ln -s ${super.tsc}/share/emacs/site-lisp/elpa/${super.tsc.name}/* \
+        $out/share/emacs/site-lisp/elpa/${esuper.pname}-${esuper.version}/
```

Comment:
This looks like wrong, however from what I understand `tsc` (`tree-sitter-core`) is supposed to be included inside the `tree-sitter` package, however we use a mostly custom packaging of `tree-sitter` (see https://github.com/NixOS/nixpkgs/pull/150239), so maybe they just missed including `tsc` in main `tree-sitter` package. This shouldn't be a problem for most users in `emacsWithPackages`, because they can simply include both, but it is a problem for us since doom-emacs doesn't declare `tsc` as a dependency. Now, for some reason simply adding `(package! tsc)` to doom-emacs doesn't solve the problem either, and I am assuming it is a weird dependency problem here.

For now, this workaround is slightly ugly but it works. In the future I will see if I can work to improve the `tree-sitter` package itself in `nixpkgs`.